queries/snakemake/injections.scm

```diff
@@ -0,0 +1,3 @@
+; inherits: python
+
+(wildcard (constraint) @regex)
```

Comment:
Neovim uses the upstream format for injections now (see CONTRIBUTING.md). You need to rebase on (not merge!) the latest `master`.